app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AdminQueueNewsletterActionGroup.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+
+    <actionGroup name="AdminQueueNewsletterActionGroup">
+        <annotations>
+            <description>Sends Newsletter template to queue:
```

Comment:
Let me shed some additional light using this example. 
In HTML/XHTML/XML the most commonly used practices in terms of such indentations are the following.

If the opening and closing tags are in the same line, we may put content between these tags.

```xml
<tag>Content</tag>
```

If we want to split lines, the common practice is to have the opening and closing tags in dedicated lines.

```xml
<tag>
    Some long long long long content
</tag>
```

So having

```xml
<tag> Some long long long long 
    and even more long content
</tag>
```

Is not a common practice.

I hope this example makes sense. 

Thank you!

Reply:
Has been fixed.